samples/web/content/apprtc/room.py

```diff
@@ -65,8 +81,70 @@ def has_room(host, room_id):
   room = memcache_client.get(key)
   return room is not None
 
+def get_room(host, room_id):
+  key = get_memcache_key_for_room(host, room_id)
+  memcache_client = memcache.Client()
+  return memcache_client.get(key)
+
+def get_room_state(host, room_id):
+  room = get_room(host, room_id)
+  if room is None:
+    return None
+  return room.get_room_state()
+
+def remove_room_for_declined_call(host, room_id, callee_gcm_id):
+  memcache_client = memcache.Client()
+  key = get_memcache_key_for_room(host, room_id)
+  
+  retries = 0
+  while retries < constants.ROOM_MEMCACHE_RETRY_LIMIT:
```

Comment:
nit: can avoid having to manually increment by doing:
for retries in xrange(constants.ROOM_MEMCACHE_RETRY_LIMIT):